paasta_tools/dump_locally_running_services.py

```diff
@@ -25,14 +25,16 @@
 import argparse
 import json
 import sys
+from typing import List
+from typing import Tuple
 
 from paasta_tools.kubernetes_tools import get_kubernetes_services_running_here_for_nerve
-from paasta_tools.marathon_tools import get_marathon_services_running_here_for_nerve
+from paasta_tools.long_running_service_tools import ServiceNamespaceConfig
 from paasta_tools.marathon_tools import get_puppet_services_running_here_for_nerve
 from paasta_tools.utils import DEFAULT_SOA_DIR
 
 
-def parse_args(argv):
+def parse_args(argv) -> argparse.Namespace:
```

Comment:
```suggestion
def parse_args(argv: Optional[Sequence[str]]) -> argparse.Namespace:
```

if you'd like to be extra thorough :p

Reply:
(would be the same typing for `main()` too)